### DISCUSSION TITLE
hosting laravel orchid on xampp

### DISCUSSION THREAD

**Discussion by shah180** [orchidsoftware/platform#1567] [Q&A] [answered]
Hi,

I have installed laravel orchid project inside C:\xampp\htdocs\orchid-project. However, I am not able to access the login page. I have also hosted other projects inside the htdocs. For example, I have another project called services in the htdocs and If I access

http://localhost/services/, then the index.phpin the service project is loaded. However, nothing like that is appearing for the orchid project. The documentation says that if I access http://localhost:8000/admin, then login page should appear. However, since i am hosting it on xampp inside htdocs and it is not on root, I tried accessing http://localhost/orchid-project/admin but nothing loads and I get an error that the url was not found

Can someone please guide here

Regards
Syed

**Comment by blankster** [comment]
Did you set the APP_URL in .env correctly (http://localhost/services in your case)?

**Reply by shah180** [reply]
Yes, I have set it to APP_URL=http://localhost/orchid-project

**Reply by blankster** [reply]
Try http://localhost/orchid-project/public/admin/ in your browser - that should work 🙂

**Reply by blankster** [reply]
Explanation: Normally "public" is your domain root. As you're using it as sub-directory, you need to enter the full URL.

Bonus:

If you decide to use it as virtual-host in another directory than a subdirectory of httpdocs, e.g. http://orchid-project.local then the following should help (of course you need to adapt the paths according to your system):

```
<VirtualHost orchid-project.local:80>
    ServerAdmin youremail@gmail.com
    DocumentRoot "C:/Users/blankster/Documents/Projects/orchid-project/public"
    ServerName orchid-project.local
    <Directory "C:/Users/blankster/Documents/Projects/orchid-project/public">
        Options Indexes FollowSymLinks Includes ExecCGI
        AllowOverride All
        Require all granted
    </Directory>
    ErrorLog "logs/orchid-project.local-error.log"
    CustomLog "logs/orchid-project.local-access.log" common
</VirtualHost>
```

Don't forget:
* Restart apache
* Edit your hosts file

Afterwards, you can access it at http://orchid-project.local/admin 🙂

**Comment by shah180** [comment]
Ok, I found that I have to access it like this. http://localhost/orchid/public/admin/. I have to figure out how to make the url to  http://localhost/orchid/

**Reply by blankster** [reply]
Well that's what I wrote above - I'm glad that it worked 🙂

If you want to change the "admin" part of the URL, check https://orchid.software/en/docs/configuration/#platform-prefix

For example:

`'prefix' => env('DASHBOARD_PREFIX', '/orchid'),`

You can access your admin-dashboard then by http://localhost/orchid-project/public/orchid/

If you want to have it exactly accessible as http://localhost/orchid, then you need to change your DocumentRoot in your apache configuration to 

`DocumentRoot "/xampp/htdocs/orchid-project/public/"`

Then you can access it by http://localhost/orchid/ . I woulnd't recommend you to change DocumentRoot of your xampp installation and better use a virtual-host.

**Reply by shah180** [reply]
Thanks. It worked

**Reply by shah180** [reply]
Yes, your answer has better explanation and more detail  and I was able to make the URL shorter using your tips. I am not able to mark your reply as an answer but many thanks for your quick answers

**Reply by blankster** [reply]
You're welcome 🙂

**Comment by blankster** [comment, marked as answer]
As a workaround, as sub-comments aren't possible to mark as a solution, you can mark this one, that references the two solutions:

* [How to access Orchid on xampp](https://github.com/orchidsoftware/platform/discussions/1567#discussioncomment-373157)
* [How to customize Orchid's URL](https://github.com/orchidsoftware/platform/discussions/1567#discussioncomment-373339 )

I guess that should work 🙂

**Comment by hellodit** [comment]
Hi @shah180 for easy use, I recommend [Laragon ](https://laragon.org/)
